src/front/spv/mod.rs

```diff
@@ -2039,15 +2014,13 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                 .future_member_decor
                 .remove(&(id, i))
                 .unwrap_or_default();
-            let origin = decor.get_origin()?;
             members.push(crate::StructMember {
                 name: decor.name,
-                origin,
+                span: None, //TODO
```

Comment:
potential to note here if this might be related to https://github.com/gfx-rs/naga/issues/336 or some other leading instructions, also there are other `span: None` data types not flagged `//TODO` i.e. in changes to `src/front/glsl/parser.rs`